### ISSUE TITLE
Binary relation link

### ISSUE THREAD

**Issue by haelexuis** [phpmyadmin/phpmyadmin#14780]
### Describe the bug

When I use binary(16) instead of ID there is no way I can see referenced row in foreign table, when I click to binary value, nothing happends, and link is

http://localhost/pma/tbl_get_field.php?db=store&table=configuration_translation&where_clause=%60configuration_translation%60.%60id%60+%3D+CAST%280x33e0163b7e8348c0b2b07addb28aab27+AS+BINARY%29&transform_key=entity_id&sql_query=SELECT+%2A+FROM+%60configuration_translation%60

Destination table is configuration, because there is reference in configuration_translation (entity column as seen on screen)

### To Reproduce

1. Go to your database
2. Create 2 tables with PK byte(16) and set up reference (OneToMany for example)
3. Insert row in 1. table with reference column byte(16) to second table
4. When you see inserted row and try to click on column with byte(16) refferal, nothing happend, and when you try to open it in new window, you just downloaded .bin file.

### Expected behavior

Show 1 selected row in destination table

### Screenshots

![image](https://user-images.githubusercontent.com/9253908/49930462-f3ac4a80-fec4-11e8-955e-52c70878a0d1.png)

### Server configuration

- Operating system: Ubuntu 18.04
- Web server: Nginx 1.14.0
- Database version: MariaDB 10.1.34
- PHP version: 7.2.10
- phpMyAdmin version: 4.8.4 / 5.0.0

### Client configuration

- Browser: Chrome
- Operating system: Windows 10

**Comment by williamdes** [comment]
@haelexuis I agree, for me downloading the `.bin` file is useless (on foreign key links)

@ibennetch Should we change foreign links to go to destination and keep the hex download on a non foreign key field ?

`QA_4_8` or `master` ?

**Comment by Rixafy** [comment]
Should I check out codebase and create a PR or you are planning to do it? @williamdes

**Comment by williamdes** [comment]
@Rixafy You can create a PR from `master` branch to `master` branch ;)

**Comment by AndriesVDW** [comment]
Is anyone currently working on this issue?

**Comment by Rixafy** [comment]
I'm currently little busy with work, so I didn't have time to even find it in the codebase

**Comment by williamdes** [comment]
Oh waw, this is a so old issue. Assigning it to 5.2

**Comment by williamdes** [comment]
Wow, sorting this out
It's a bug I think!
Probably this is what I described on https://github.com/phpmyadmin/phpmyadmin/issues/16013#issuecomment-1283850036

I really want this old one to be fixed

If someone has feedback to confirm this is what I discovered while testing?
